web_console/customizing-the-web-console.adoc

```diff
@@ -17,6 +17,13 @@ include::modules/creating-custom-links.adoc[leveloffset=+1]
 
 include::modules/customizing-the-web-console-URL.adoc[leveloffset=+1]
 
+[id="customizing-web-console-recognize"]
+== Recognizing resource and project limits and quotas
+
+When in Developer mode of the web console, when in the *Topology* view, a graphical representation of available resources appears. A yellow border appears around the resource name indicating a message regarding resource limitation or quotas being reached. Click the resource to open a side panel to see the message. If the *Topology* view has been zoomed out, a yellow dot indicates that a message is provided.
```

Comment:
Please use "**Developer** perspective" instead of "Developer mode" here to keep consistency with other docs about dev console. See https://docs.openshift.com/container-platform/4.13/applications/creating_applications/odc-creating-applications-using-developer-perspective.html for examples

Reply:
done